Lodge Map Rework

Rat-Time opened the pull request:
Lodge map rework.  No more underground, all interesting cave features have been migrated to the surface along with materials and rewards useful to lodge hunters.
The endless xenos were removed, as was the greyson facility.  Sargoyles and tribal birds have an increased presence and their own locations, as well as lairs for the solitary large tengolo.  

![StrongDMM_uoTMMkbStS](https://github.com/user-attachments/assets/23510378-a337-4c31-811e-371be5a32775)

RagingLemons1 commented:
RIP lodge greysons, I never got to shoot a hunter asking why we were raiding (((their))) dungeon

Rat-Time commented:
Added a greyson facility, moved the renders into the xeno cave and the xenos into the larger render cave.
![StrongDMM_XzkHZospKr](https://github.com/user-attachments/assets/b039e7f2-67c5-4203-a473-ba418e8efdb4)
![StrongDMM_od4C69Nd1d](https://github.com/user-attachments/assets/68ba0e1d-176c-4376-9788-11bec21f7ea9)
![StrongDMM_mxdFFihYYL](https://github.com/user-attachments/assets/dcbe84d2-e599-4cb0-b53b-51dc67023bf5)
